src/Components/CreateImageWizard/steps/Oscap/index.tsx

```diff
@@ -26,24 +49,78 @@ const OscapStep = () => {
   return (
     <Form>
       <Title headingLevel="h1" size="xl">
-        OpenSCAP profile
+        {complianceEnabled ? 'Compliance' : 'OpenSCAP profile'}
       </Title>
-      <Text>
-        OpenSCAP enables you to automatically monitor the adherence of your
-        registered RHEL systems to a selected regulatory compliance profile.
-        <br />
-        <Button
-          component="a"
-          target="_blank"
-          variant="link"
-          icon={<ExternalLinkAltIcon />}
-          iconPosition="right"
-          isInline
-          href={COMPLIANCE_AND_VULN_SCANNING_URL}
-        >
-          Documentation
-        </Button>
-      </Text>
+      {complianceEnabled && (
+        <FormGroup>
+          <Radio
+            id="openscap radio openscap type"
+            label="OpenSCAP"
+            name="oscap-radio-openscap"
+            isChecked={complianceType === 'openscap'}
+            onChange={() =>
+              dispatch(changeComplianceType('openscap' as ComplianceType))
```

Comment:
Should the data selected in the other option also get dropped when switching between compliance/openscap? Currently it's possible to select compliance with openscap profile selected.

Reply:
Yea so I wondered about that and I'm not sure. Like currently it'll grab the profile that the policy was based on when switching from openscap to compliance. When switching back without changing the profile the policy shuold still be there. I'm not sure, for the file system customisation we do something where we keep the mountpoints, so when the user switches back and forth they're preserved.

Reply:
Hmm, I see. Why I'm asking is this case - I've picked an Oscap profile and switched to Compliance. The Oscap profile information is still there, I guess what confuses me is - are we handling oscap and compliance as two separate things or are we looking at compliance like more of a complement to originally selected oscap profile? Hope that's not a super stupid question, still trying to figure out how it works :sweat_smile: 

![image](https://github.com/user-attachments/assets/4a9cb813-0319-478a-839f-075cf9b84ee8)

Reply:
@croissanne Just this question, otherwise I'd say all good :+1:

Reply:
Ah right, yea that is a tricky one, the oscap profile information simply takes whatever profile id is present. Maybe it's best to clear between states for now?

Reply:
Perfect, makes sense to me. At least for now, we can refine things as we go. One last thing - when clearing between states we'll need to also clear oscap packages, fsc, kernel and services so those don't get left behind ig.

So something like `handleClear()` in `src/Components/CreateImageWizard/steps/Oscap/Oscap.tsx` maybe?

Reply:
oh right, good point

Reply:
hmm there's definitely some duplication going on now, but moving the radio into oscap would require more restructuring.

Reply:
A bit of duplication is okay, we can revisit it in a follow up :+1: